frontend/cypress/integration/authRedirects.spec.js

```diff
@@ -0,0 +1,28 @@
+import faker from 'faker'
+
+describe('Validates authentication redirects', () => {
+  const username = faker.name.firstName()
+  const email = faker.internet.email()
+  const password = '123456'
+
+  before(() => {
+    cy.visit('/')
+    cy.createUser(username, email, password)
+  })
+  it("Redirects unauthenticated users to '/login' when accessing reqSignIn page", () => {
+    // `/project/new` is marked as `reqSignIn`.
+    cy.visit('/projects/new')
+    cy.url().should('eq', 'http://kitspace.test:3000/login')
```

Comment:
Can we add a test that it re-directs back to the requested page (`/projects/new`) after login?